cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_builder/migration.spec.js

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  BASE_PATH,
+  toTestId,
+  VB_APP_PATH,
+  VB_INDEX_ID,
+  VB_METRIC_VIS_TITLE,
+  VB_PATH_INDEX_DATA,
+  VISUALIZATION_PATH_SO_DATA,
+} from '../../../../../utils/constants';
+import { CURRENT_TENANT } from '../../../../../utils/commands';
+
+if (Cypress.env('VISBUILDER_ENABLED')) {
+  describe('Vis Builder: Metric Chart', () => {
+    before(() => {
+      CURRENT_TENANT.newTenant = 'global';
+      cy.fleshTenantSettings();
+      cy.deleteIndex(VB_INDEX_ID);
+      cy.bulkUploadDocs(VB_PATH_INDEX_DATA);
+      cy.importSavedObjects(VISUALIZATION_PATH_SO_DATA);
+    });
+
+    beforeEach(() => {
+      CURRENT_TENANT.newTenant = 'global';
+      cy.fleshTenantSettings();
+    });
+
+    it('Show existing visualizations in Visualize and navigate to it', () => {
+      cy.visit(`${BASE_PATH}/app/visualize`);
+      cy.get('input[type="search"]').type(`${VB_METRIC_VIS_TITLE}{enter}`);
+      cy.get('.euiBasicTable-loading').should('not.exist'); // wait for the loading to stop
```

Comment:
potential action item:

does euiBasicTable-loading have a test id in the page we use it? if not we could probably add one.